### ISSUE TITLE
Add Guidestar rating to Donations page

### ISSUE THREAD

**Issue by masinter** [Interlisp/medley#1720]
```
<a href="https://www.guidestar.org/profile/shared/31fad8e3-de97-4d71-9401-b896bf085498" target="_blank"><img src="https://widgets.guidestar.org/TransparencySeal/10019973" /></a>
```

FYI https://www.guidestar.org/Profile/ViewCCPdf?ein=87-2528093

**Comment by pamoroso** [comment]
When visiting https://www.guidestar.org/Profile/ViewCCPdf?ein=87-2528093 I get the error "You do not have access to the requested page".

**Comment by stumbo** [comment]
Maybe go with this link:  [https://www.guidestar.org/profile/87-2528093](https://www.guidestar.org/profile/87-2528093)?

**Comment by pamoroso** [comment]
Addressed by [PR #228](https://github.com/Interlisp/Interlisp.github.io/pull/228).
